### ISSUE TITLE
Amp-carousel sometimes won't load images. 

### ISSUE THREAD

**Issue by lvberkum** [ampproject/amphtml#23128]
## What's the issue?

Amp-carousel sometimes won't load images if a page is called without specifying its (default) filename.

## How do we reproduce the issue?

https://www.homes-in-provence.com/ => does not always load carousel images. Always loads carousel HTML though. Images won't load 2 out of 10 reloads on average.

https://www.homes-in-provence.com/index.html => always loads carousel images.

## What browsers are affected?

FireFox always. Chome sometimes.

## Which AMP version is affected?

This is a new issue, first noticed on or around June 22nd. Amp version 1906191709480.

**Comment by cathyxz** [comment]
Hmm. I took a quick look at this in both Firefox and Chrome and I'm observing that the carousel images do load 10/10 times (with network throttling as well). Could I ask for a few more clarifications? 

Are you referring to this carousel on the page or something else? 
<img width="1251" alt="Screenshot 2019-07-02 at 12 30 37 PM" src="https://user-images.githubusercontent.com/1528181/60541040-7bbd3a80-9cc5-11e9-9c9f-3a6635df23fd.png">
<img width="634" alt="Screenshot 2019-07-02 at 12 31 07 PM" src="https://user-images.githubusercontent.com/1528181/60541041-7c55d100-9cc5-11e9-9208-a0377105f17f.png">
Are you on a mobile or desktop device? 
I'm attempting to repro on a mac, are you on Windows device by any chance?

**Comment by lvberkum** [comment]
That carousel as well, but it is maybe easier to focus on the first cover image carousel, which loops three 960x480 images at the top of the page.

I just checked again, Amsterdam 8:30am. FireFox on Mac and Windows on desktop still affected. Safari on mobile as well. Chrome seems to be ok on desktop and mobile.  

I checked from various locations including the Netherlands, Germany, Spain and United States (Los Angeles). All locations are affected.

I've included a network monitor screenshot which shows the delayed loading of the first image of the first carousel when the problem occurs. It only loads after 30 seconds. Other carousel images do not load at all.

The problem only occurs when loading index.html implicitly, no problems loading index.html directly. (I've also included a network monitor screenshot of the direct load.)

![HomesInProvence_NetworkMonitor_implicit_load](https://user-images.githubusercontent.com/25842139/60569828-7ff26200-9d70-11e9-8351-3b82cff5e828.PNG)

![HomesInProvence_NetworkMonitor_direct_load](https://user-images.githubusercontent.com/25842139/60569866-98627c80-9d70-11e9-8e0f-07e70cb79dfd.PNG)

**Comment by cathyxz** [comment]
Thank you! I'll take another look.

**Comment by lvberkum** [comment]
Hi! I cannot reproduce the problem anymore, since a few days actually, I kept this open in case the problem would reappear. Any insight you might have would be more than welcome!

**Comment by cathyxz** [comment]
Still can't repro unfortunately, feel free to reopen if the issue reoccurs.
